<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
  <title>CRAN Task View: Meta-Analysis</title>
  <link rel="stylesheet" type="text/css" href="../CRAN_web.css" />
  <meta http-equiv="content-type" content="text/html; charset=UTF-8" />
  <meta name="citation_title" content="CRAN Task View: Meta-Analysis" />
  <meta name="citation_author" content="Michael Dewey" />
  <meta name="citation_publication_date" content="2018-05-10" />
  <meta name="citation_public_url" content="https://CRAN.R-project.org/view=MetaAnalysis" />
  <meta name="DC.title" content="CRAN Task View: Meta-Analysis" />
  <meta name="DC.creator" content="Michael Dewey" />
  <meta name="DC.issued" content="2018-05-10" />
  <meta name="DC.identifier" content="https://CRAN.R-project.org/view=MetaAnalysis" />
</head>

<body>
  <h2>CRAN Task View: Meta-Analysis</h2>
  <table summary="MetaAnalysis task view information">
    <tr><td valign="top"><b>Maintainer:</b></td><td>Michael Dewey</td></tr>
    <tr><td valign="top"><b>Contact:</b></td><td>&#x6c;&#x69;&#x73;&#x74;&#x73;&#x20;&#x61;&#x74;&#x20;&#x64;&#x65;&#x77;&#x65;&#x79;&#x2e;&#x6d;&#x79;&#x7a;&#x65;&#x6e;&#x2e;&#x63;&#x6f;&#x2e;&#x75;&#x6b;</td></tr>
    <tr><td valign="top"><b>Version:</b></td><td>2018-05-10</td></tr>
    <tr><td valign="top"><b>URL:</b></td><td><a href="https://CRAN.R-project.org/view=MetaAnalysis">https://CRAN.R-project.org/view=MetaAnalysis</a></td></tr>
  </table>

    <div>
      <p>
        This task view covers packages which include
   facilities for meta-analysis
   of summary statistics from primary studies.
   The task view does not consider
   the meta-analysis of individual participant data (IPD)
   which can be handled by
   any of the standard linear modelling functions
   but does include some
   packages which offer special facilities for IPD.
      </p>
      <p>
        The standard meta-analysis model is a form of
  weighted least squares and so
  any of the wide range of R packages providing
  weighted least squares would
  in principle be able to fit the model.
  The advantage of using a specialised package is
  that (a) it takes care of the small tweaks necessary
  (b) it provides a range
  of ancillary functions for displaying
  and investigating the model.
  Where the model is referred to below it is this
  model which is meant.
      </p>
      <p>
        Where summary statistics are not available
  a meta-analysis of significance
  levels is possible.
  This is not completely unconnected with the problem
  of adjustment for multiple comparisons but
  the packages below which offer this,
  chiefly in the context of genetic data,
  also offer additional functionality.
      </p>
      <h4>
        Univariate meta-analysis
      </h4>
      <p>
        <i>
          Preparing for meta-analysis
        </i>
      </p>
      <ul>
        <li>
          The primary studies often use a range of
    statistics to present their
    results.
    Convenience functions to convert these onto a common
    metric are presented by:
<a href="../packages/compute.es/index.html">compute.es</a>
          which converts from
    various statistics to
    d, g, r, z and the log odds ratio,
<a href="../packages/MAc/index.html">MAc</a>
          which converts to correlation coefficients,
<a href="../packages/MAd/index.html">MAd</a>
          which converts to mean differences,
    and
<a href="../packages/metafor/index.html">metafor</a>
          which converts to effect sizes
    an extensive set of measures
    for comparative studies (such as binary data,
    person years, mean differences and
    ratios and so on), for studies of association
    (a wide range of correlation types), for non-comparative
    studies (proportions, incidence rates, and mean change).
    It also provides for a measure 
    used in psychometrics (Cronbach's alpha).
<a href="../packages/esc/index.html">esc</a>
          provides
   a range of effect size calculations with partial overlap
   with
<a href="../packages/metafor/index.html">metafor</a>
          but with some extras, noticeably
   for converting test statistics, also includes a
   convenience function for collating
   its output for input to another
   package like
<a href="../packages/metafor/index.html">metafor</a>
          or producing a CSV file.
<a href="../packages/effsize/index.html">effsize</a>
          contains functions to compute effect sizes mean difference (Cohen's
   d and Hedges g), dominance matrices (Cliff's Delta)
   and stochastic superiority (Vargha-Delaney A).
<a href="../packages/psychmeta/index.html">psychmeta</a>
          provides extensive facilties for
   converting effect sizes and for correcting for a variety
   of restrictions and measurement errors.
        </li>
        <li>
<a href="../packages/meta/index.html">meta</a>
          provides functions to read and work
    with files output by RevMan 4 and 5.
        </li>
        <li>
<a href="../packages/metagear/index.html">metagear</a>
          provides many tools for the
    systematic review process including screening articles,
    downloading the articles, generating a PRISMA diagram,
    and some tools for effect sizes.
<a href="../packages/revtools/index.html">revtools</a>
          provides tools for downloading from bibliographic
    databases and uses machine learning methods to process them.
        </li>
        <li>
<a href="../packages/metavcov/index.html">metavcov</a>
          computes the variance-covariance
   matrix for multivariate meta-analysis
   when correlations between outcomes can be
   provided but not between treatment effects, and
<a href="../packages/clubSandwich/index.html">clubSandwich</a>
          imputes
  variance-covariance matrix for multivariate meta-analysis
        </li>
        <li>
<a href="../packages/metafuse/index.html">metafuse</a>
          uses a fused lasso to merge
    covariate estimates across a number of independent datasets.
        </li>
      </ul>
      <p>
        <i>
          Fitting the model
        </i>
      </p>
      <ul>
        <li>
          Four packages provide the inverse variance weighted,
    Mantel-Haenszel,
    and Peto methods:
<a href="../packages/epiR/index.html">epiR</a>,
<a href="../packages/meta/index.html">meta</a>,
<a href="../packages/metafor/index.html">metafor</a>, and
<a href="../packages/rmeta/index.html">rmeta</a>.
        </li>
        <li>
          For binary data
<a href="../packages/metafor/index.html">metafor</a>
          provides
    the binomial-normal model.
        </li>
        <li>
          For sparse binary data
<a href="../packages/exactmeta/index.html">exactmeta</a>
          provides an exact method which
    does not involve continuity corrections.
        </li>
        <li>
          Packages which work with specific effect sizes
    may be more congenial
    to workers in some areas of science and include
<a href="../packages/MAc/index.html">MAc</a>
          and
<a href="../packages/metacor/index.html">metacor</a>
          which provide meta-analysis of correlation
    coefficients and
<a href="../packages/MAd/index.html">MAd</a>
          which provides meta-analysis
    of mean differences.
<a href="../packages/MAc/index.html">MAc</a>
          and
<a href="../packages/MAd/index.html">MAd</a>
          provide
    a range of graphics.
<a href="../packages/psychometric/index.html">psychometric</a>
          provides an extensive range of functions
    for the meta-analysis of psychometric studies.
        </li>
        <li>
<a href="../packages/psychmeta/index.html">psychmeta</a>
          implements the Hunter-Schmidt method
  including corrections for reliability and range-restriction issues
        </li>
        <li>
          Bayesian approaches are contained in various packages.
<a href="../packages/bspmma/index.html">bspmma</a>
          which 
    provides two different models:
    a non-parametric and a semi-parametric.
    Graphical display of the results is provided.
<a href="../packages/metamisc/index.html">metamisc</a>
          provides a method
    with priors suggested by Higgins.
<a href="../packages/mmeta/index.html">mmeta</a>
          provides meta-analysis using
    beta-binomial prior distributions.
   A Bayesian approach is also provided by
<a href="../packages/bmeta/index.html">bmeta</a>
          which
   provides forest plots via
<a href="../packages/forestplot/index.html">forestplot</a>
          and diagnostic graphical output.
<a href="../packages/bayesmeta/index.html">bayesmeta</a>
          includes shrinkage estimates, posterior
    predictive p-values and forest plots via either
<a href="../packages/metafor/index.html">metafor</a>
          or
<a href="../packages/forestplot/index.html">forestplot</a>. Diagnostic graphical output is available.
        </li>
        <li>
          Some packages concentrate on providing
    a specialised version of the core
    meta-analysis function without providing
    the range of ancillary
    functions. These are:
<a href="../packages/gmeta/index.html">gmeta</a>
          which subsumes a very wide variety of models under the method
   of confidence distributions and
   also provides a graphical display,
<a href="../packages/metaLik/index.html">metaLik</a>
          which uses a more sophisticated approach
    to the likelihood,
<a href="../packages/metamisc/index.html">metamisc</a>
          which as well as the
    method of moments provides
    two likelihood-based  methods, and
<a href="../packages/metatest/index.html">metatest</a>
          which provides
    another improved method of obtaining confidence intervals,
<a href="../packages/metaBMA/index.html">metaBMA</a>
          has a
    Bayesian approach using model averaging, a variety of priors
    are provided and it is possible for the user to define
    new ones.
        </li>
        <li>
<a href="../packages/metagen/index.html">metagen</a>
          provides a range of methods for
    random effects models and also facilities
    for extensive simulation studies of the
    properties of those methods.
        </li>
        <li>
<a href="../packages/metaplus/index.html">metaplus</a>
          fits random effects
    models relaxing the usual
    assumption that the random effects have a normal
    distribution by providing t or a mixture
    of normals.
        </li>
        <li>
<a href="../packages/ratesci/index.html">ratesci</a>
          fits random effects models to binary data using
   a variety of methods for confidence intervals.
        </li>
        <li>
<a href="../packages/RandMeta/index.html">RandMeta</a>
          estimates exact confidence intervals in random effects
   models using an efficient algorithm.
        </li>
        <li>
<a href="../packages/rma.exact/index.html">rma.exact</a>
          estimates exact confidence intervals in random effects
   normal-normal models and also provides plots of them.
        </li>
        <li>
<a href="../packages/clubSandwich/index.html">clubSandwich</a>
          gives cluster-robust variance estimates.
        </li>
        <li>
<a href="../packages/pimeta/index.html">pimeta</a>
          implements prediction intervals for random effects meta-analysis.
        </li>
        <li>
<a href="../packages/RBesT/index.html">RBesT</a>
          generates a meta-analytic prior for future studies.
   Forest plots are available as well as diagnostic plots.
   It uses Stan as its engine.
        </li>
      </ul>
      <i>
        Graphical methods
      </i>
      <p>
        An extensive range of graphical procedures is available.
      </p>
      <ul>
        <li>
          Forest plots are provided in
<a href="../packages/forestmodel/index.html">forestmodel</a>
          (using ggplot2),
<a href="../packages/forestplot/index.html">forestplot</a>,
<a href="../packages/meta/index.html">meta</a>,
<a href="../packages/metafor/index.html">metafor</a>,
<a href="../packages/psychmeta/index.html">psychmeta</a>, and
<a href="../packages/rmeta/index.html">rmeta</a>.
    Although the most basic plot can be produced
    by any of them
    they each provide their own choice of enhancements.
        </li>
        <li>
          Funnel plots are provided in
<a href="../packages/meta/index.html">meta</a>,
<a href="../packages/metafor/index.html">metafor</a>,
<a href="../packages/psychometric/index.html">psychometric</a>
          and
<a href="../packages/rmeta/index.html">rmeta</a>.
    In addition to the standard funnel plots
    an enhanced funnel plot to assess the
    impact of extra evidence
    is available in
<a href="../packages/extfunnel/index.html">extfunnel</a>, a funnel plot
    for limit meta-analysis in
<a href="../packages/metasens/index.html">metasens</a>, and
<a href="../packages/metaviz/index.html">metaviz</a>
          provides
    funnel plots in the context of visual inference.
        </li>
        <li>
          Radial (Galbraith) plots are provided in
<a href="../packages/meta/index.html">meta</a>
          and
<a href="../packages/metafor/index.html">metafor</a>.
        </li>
        <li>
          L'Abbe plots are provided in
<a href="../packages/meta/index.html">meta</a>
          and
<a href="../packages/metafor/index.html">metafor</a>.
        </li>
        <li>
          Baujat plots are provided in
<a href="../packages/meta/index.html">meta</a>
          and
<a href="../packages/metafor/index.html">metafor</a>.
        </li>
        <li>
<a href="../packages/metaplotr/index.html">metaplotr</a>
          provides a crosshair plot
        </li>
        <li>
<a href="../packages/MetaAnalyser/index.html">MetaAnalyser</a>
          provides an interactive
   visualisation of the results of a meta-analysis.
        </li>
        <li>
<a href="../packages/metaviz/index.html">metaviz</a>
          provides rainforestplots, an
   enhanced version of forest plots. It accepts
   input from
<a href="../packages/metafor/index.html">metafor</a>.
        </li>
      </ul>
      <i>
        Investigating heterogeneity
      </i>
      <ul>
        <li>
          Confidence intervals for the heterogeneity parameter
    are provided in
<a href="../packages/metafor/index.html">metafor</a>,
<a href="../packages/metagen/index.html">metagen</a>, and
<a href="../packages/psychmeta/index.html">psychmeta</a>.
        </li>
        <li>
<a href="../packages/altmeta/index.html">altmeta</a>
          presents a variety of alternative methods for measuring
  and testing heterogeneity with a focus on robustness
  to outlying studies.
        </li>
        <li>
<a href="../packages/hetmeta/index.html">hetmeta</a>
          calculates some extra measures of heterogeneity.
        </li>
        <li>
<a href="../packages/metaforest/index.html">metaforest</a>
          investigates heterogeneity using random forests.
  Note that it has nothing to do with forest plots.
        </li>
      </ul>
      <i>
        Model criticism
      </i>
      <ul>
        <li>
          An extensive series of plots of diagnostic statistics is
    provided in
<a href="../packages/metafor/index.html">metafor</a>.
        </li>
        <li>
<a href="../packages/metaplus/index.html">metaplus</a>
          provides outlier diagnostics.
        </li>
        <li>
<a href="../packages/psychmeta/index.html">psychmeta</a>
          provides leave-one-out methods.
        </li>
        <li>
<a href="../packages/ConfoundedMeta/index.html">ConfoundedMeta</a>
          conducts a sensitivity analysis
   to estimate the proportion of studies with
   true effect sizes above a threshold.
        </li>
      </ul>
      <i>
        Investigating small study bias
      </i>
      <p>
        The issue of whether small studies give different results
    from large studies has been addressed by visual
    examination of  the funnel plots mentioned above.
    In addition:
      </p>
      <ul>
        <li>
<a href="../packages/meta/index.html">meta</a>
          and
<a href="../packages/metafor/index.html">metafor</a>
          provide
    both the non-parametric method suggested
    by Begg and Mazumdar
    and a range of regression tests modelled
    after the approach of Egger.
        </li>
        <li>
<a href="../packages/xmeta/index.html">xmeta</a>
          provides a method in the context of
  multivariate meta-analysis.
        </li>
        <li>
          An exploratory technique for detecting
    an excess of statistically
    significant studies is provided by
<a href="../packages/PubBias/index.html">PubBias</a>.
        </li>
        <li>
<a href="../packages/metamisc/index.html">metamisc</a>
          provides funnel plots and tests for asymmetry.
        </li>
      </ul>
      <i>
        Unobserved studies
      </i>
      <p>
        A recurrent issue in meta-analysis has been
    the problem of unobserved studies.
      </p>
      <ul>
        <li>
          Rosenthal's fail safe n is provided by
<a href="../packages/MAc/index.html">MAc</a>
          and
<a href="../packages/MAd/index.html">MAd</a>.
<a href="../packages/metafor/index.html">metafor</a>
          provides it as well as two
    more recent methods  by Orwin and Rosenberg.
        </li>
        <li>
          Duval's trim and fill method is provided
    by
<a href="../packages/meta/index.html">meta</a>
          and
<a href="../packages/metafor/index.html">metafor</a>.
        </li>
        <li>
<a href="../packages/metasens/index.html">metasens</a>
          provides Copas's selection
    model and also
    the method of limit meta-analysis (a regression based
    approach for dealing with small study effects)
    due to RÃ¼cker et al.
        </li>
        <li>
<a href="../packages/selectMeta/index.html">selectMeta</a>
          provides various selection models:
    the parametric model of Iyengar and Greenhouse,
    the non-parametric model of Dear and Begg, and
    proposes a new non-parametric method imposing a 
    monotonicity constraint.
        </li>
        <li>
<a href="../packages/SAMURAI/index.html">SAMURAI</a>
          performs a sensitivity
    analysis assuming
    the number of unobserved studies is known,
    perhaps from a trial registry, but not their outcome.
        </li>
        <li>
          The
<a href="../packages/metansue/index.html">metansue</a>
          package allows the inclusion
   by multiple imputation
   of studies known only to have a non-significant
   result.
        </li>
        <li>
<a href="../packages/weightr/index.html">weightr</a>
          provides
  facilities for using the weight function model
  of Vevea and Hedges.
        </li>
      </ul>
      <i>
        Other study designs
      </i>
      <ul>
        <li>
<a href="../packages/SCMA/index.html">SCMA</a>
          provides single case meta-analysis.
    It is part of a suite of packages
    dedicated to single-case designs.
        </li>
        <li>
<a href="../packages/joint.Cox/index.html">joint.Cox</a>
          provides facilities for
	the meta-analysis of studies of joint time-to-event
	and disease progression.
        </li>
        <li>
<a href="../packages/metamisc/index.html">metamisc</a>
          provides for meta-analysis of prognostic studies
   using the c statistic or the O/E ratio. Some plots are provided.
        </li>
      </ul>
      <i>
        Meta-analysis of significance values
      </i>
      <ul>
        <li>
<a href="../packages/metap/index.html">metap</a>
          provides some facilities for
    meta-analysis of significance values.
        </li>
        <li>
<a href="../packages/aggregation/index.html">aggregation</a>
          provides a smaller subset of methods.
        </li>
        <li>
<a href="../packages/TFisher/index.html">TFisher</a>
          provides Fisher's method using thresholding for
    the p-values.
        </li>
      </ul>
      <p>
        Some methods are also provided in some
    of the genetics packages mentioned below.
      </p>
      <h4>
        Multivariate meta-analysis
      </h4>
      <p>
        Standard methods outlined above assume that
    the effect sizes are independent.
    This assumption may be violated in a number of ways:
    within each primary study multiple treatments may
    be compared to the same control,
    each primary study may report multiple
    endpoints, or primary studies may be clustered
    for instance because they come from
    the same country or the same research team.
    In these situations where the outcome is multivariate:
      </p>
      <ul>
        <li>
<a href="../packages/mvmeta/index.html">mvmeta</a>
          assumes the within study covariances
    are known and provides a
    variety of options for fitting random effects.
<a href="../packages/metafor/index.html">metafor</a>
          provides fixed effects and likelihood
    based random effects model fitting procedures.
    Both these packages include meta-regression,
<a href="../packages/metafor/index.html">metafor</a>
          also provides for clustered and
    hierarchical models.
        </li>
        <li>
<a href="../packages/mvtmeta/index.html">mvtmeta</a>
          provides multivariate meta-analysis 
    using the method of moments for random effects
    although not meta-regression,
        </li>
        <li>
<a href="../packages/metaSEM/index.html">metaSEM</a>
          provides multivariate
    (and univariate) meta-analysis and
    meta-regression by embedding it in the
    structural equation framework
    and using OpenMx for the structural equation modelling.
    It can provide a three-level meta-analysis
    taking account of clustering and allowing for
    level 2 and level 3 heterogeneity.
    It also provides via a two-stage approach
    meta-analysis of correlation or covariance matrices.
        </li>
        <li>
<a href="../packages/xmeta/index.html">xmeta</a>
          provides various functions for multivariate meta-analysis
      and also for detecting publication bias.
        </li>
        <li>
<a href="../packages/dosresmeta/index.html">dosresmeta</a>
          concentrates on the situation
    where individual studies have information on
    the dose-response relationship.
        </li>
        <li>
<a href="../packages/robumeta/index.html">robumeta</a>
          provides robust variance
    estimation for clustered and hierarchical estimates.
        </li>
        <li>
<a href="../packages/CIAAWconsensus/index.html">CIAAWconsensus</a>
          has a function for multivariate m-a in the context
   of atomic weights and estimating
   isotope ratios.
        </li>
      </ul>
      <h4>
        Meta-analysis of studies of diagnostic tests
      </h4>
      <p>
        A special case of multivariate meta-analysis
    is the case of summarising
    studies of diagnostic tests.
    This gives rise to a bivariate, binary
    meta-analysis with the within-study correlation
    assumed zero
    although the between-study correlation is estimated.
    This is an active area of research and a variety
    of methods are available
    including what is referred to here as Reitsma's
    method, and the hierarchical summary receiver operating
    characteristic (HSROC) method.
    In many situations these are equivalent.
      </p>
      <ul>
        <li>
<a href="../packages/mada/index.html">mada</a>
          provides various descriptive statistics
     and univariate methods (diagnostic odds ratio and Lehman
     model) as well as the bivariate method due to Reitsma.
     In addition meta-regression is provided.
     A range of graphical methods is also available.
        </li>
        <li>
<a href="../packages/Metatron/index.html">Metatron</a>
          provides a method for
    the Reitsma model
    incuding the case of an imperfect reference standard.
        </li>
        <li>
<a href="../packages/metamisc/index.html">metamisc</a>
          provides the method
    of Riley which estimates a common
    within and between correlation.
    Graphical output is also provided.
        </li>
        <li>
<a href="../packages/bamdit/index.html">bamdit</a>
          provides Bayesian meta-analysis
    with a bivariate random effects model
    (using JAGS to implement the MCMC method).
    Graphical methods are provided.
        </li>
        <li>
<a href="../packages/meta4diag/index.html">meta4diag</a>
          provides Bayesian inference analysis for bivariate meta-analysis
      of diagnostic test studies and an extensive range of
      graphical methods.
        </li>
        <li>
<a href="../packages/CopulaREMADA/index.html">CopulaREMADA</a>
          uses a copula based mixed model
        </li>
        <li>
<a href="../packages/diagmeta/index.html">diagmeta</a>
          considers the case where the primary studies provide
   analysis using multiple cut-offs.
   Graphical methods are also provided.
        </li>
      </ul>
      <h4>
        Meta-regression
      </h4>
      <p>
        Where suitable moderator variables are
    available they may be included using meta-regression.
    All these packages are mentioned above, this
    just draws that information together.
      </p>
      <ul>
        <li>
<a href="../packages/metafor/index.html">metafor</a>
          provides meta-regression (multiple
    moderators are catered for).
    Various packages rely on
<a href="../packages/metafor/index.html">metafor</a>
          to
    provide meta-regression (<a href="../packages/meta/index.html">meta</a>,
<a href="../packages/MAc/index.html">MAc</a>,
    and
<a href="../packages/MAd/index.html">MAd</a>) and all three of
    these provide bubble plots.
<a href="../packages/psychmeta/index.html">psychmeta</a>
          also uses
<a href="../packages/metafor/index.html">metafor</a>.
        </li>
        <li>
<a href="../packages/bmeta/index.html">bmeta</a>,
<a href="../packages/metagen/index.html">metagen</a>,
<a href="../packages/metaLik/index.html">metaLik</a>,
<a href="../packages/metaSEM/index.html">metaSEM</a>, and
<a href="../packages/metatest/index.html">metatest</a>
          also provide meta-regression.
        </li>
        <li>
<a href="../packages/mvmeta/index.html">mvmeta</a>
          provides meta-regression
    for multivariate meta-analysis
    as do
<a href="../packages/metafor/index.html">metafor</a>
          and
<a href="../packages/metaSEM/index.html">metaSEM</a>.
        </li>
        <li>
<a href="../packages/metacart/index.html">metacart</a>
          integrates regression and classification trees
  into the meta-analysis framework for moderator selection.
        </li>
        <li>
<a href="../packages/mada/index.html">mada</a>
          provides for the
    meta-regression of diagnostic test studies.
        </li>
      </ul>
      <h4>
        Individual participant data (IPD)
      </h4>
      <p>
        Where all studies can provide individual participant data
    then software for analysis of multi-centre trials
    or multi-centre cohort studies should prove adequate
    and is outside the scope of this task view.
    Other packages which provide facilities
    related to IPD are:
      </p>
      <ul>
        <li>
<a href="../packages/ipdmeta/index.html">ipdmeta</a>
          which uses information on aggregate
    summary statistics and a covariate of interest
    to assess whether a full IPD analysis
    would have more power.
        </li>
        <li>
<a href="../packages/ecoreg/index.html">ecoreg</a>
          which is designed for ecological studies
    enables estimation of an individual level
    logistic regression from aggregate data or
    individual data.
        </li>
        <li>
<a href="../packages/surrosurv/index.html">surrosurv</a>
          evaluates failure time surrogates
    in the context of IPD meta-analysis
        </li>
      </ul>
      <h4>
        Network meta-analysis
      </h4>
      <p>
        Also known as multiple treatment comparison.
    This is a very active area of research and development.
    Note that some of the packages mentioned above
    under multivariate meta-analysis can also be
    used for network meta-analysis with
   appropriate setup.
      </p>
      <p>
        This is provided in a Bayesian framework by
<a href="../packages/gemtc/index.html">gemtc</a>,
    which acts as a front-end to BUGS
    or JAGS, and
<a href="../packages/pcnetmeta/index.html">pcnetmeta</a>,
    which uses JAGS.
<a href="../packages/nmaINLA/index.html">nmaINLA</a>
        uses integrated nested Laplace approximations
   as an alternative to MCMC.
   It provides a number of data-sets.
<a href="../packages/netmeta/index.html">netmeta</a>
        works in a frequentist framework.
    Both
<a href="../packages/pcnetmeta/index.html">pcnetmeta</a>
        and
<a href="../packages/netmeta/index.html">netmeta</a>
        provide network graphs and
<a href="../packages/netmeta/index.html">netmeta</a>
        provides a heatmap for
    displaying inconsistency and heterogeneity.
<a href="../packages/nmathresh/index.html">nmathresh</a>
        provides decision-invariant bias adjustment
   thresholds and intervals the
   smallest changes to the data that would result in a change of decision.
      </p>
      <h4>
        Genetics
      </h4>
      <p>
        There are a number of packages specialising
    in genetic data:
<a href="../packages/CPBayes/index.html">CPBayes</a>
        uses a Bayesian approach to study cross-phenotype genetic
   associations,
<a href="../packages/etma/index.html">etma</a>
        proposes a new statistical method to detect epistasis,
<a href="../packages/gap/index.html">gap</a>
        combines p-values,
<a href="../packages/getmstatistic/index.html">getmstatistic</a>
        quantifies systematic heterogeneity,
<a href="../packages/MendelianRandomization/index.html">MendelianRandomization</a>
        provides several methods for performing Mendelian randomisation
   analyses with summarised data,
<a href="../packages/MetABEL/index.html">MetABEL</a>
        provides meta-analysis of
      genome wide SNP association results,
<a href="../packages/MetaIntegrator/index.html">MetaIntegrator</a>
        provides an extensive set of functions for genetic studies,
<a href="../packages/metaMA/index.html">metaMA</a>
        provides meta-analysis of
      p-values or moderated
     effect sizes to find differentially expressed genes,
<a href="../packages/MetaPath/index.html">MetaPath</a>
        performs meta-analysis for pathway enrichment,
<a href="../packages/MetaPCA/index.html">MetaPCA</a>
        provides meta-analysis in
      the dimension reduction of genomic data,
<a href="../packages/MetaQC/index.html">MetaQC</a>
        provides objective quality control and
      inclusion/exclusion criteria for genomic meta-analysis,
<a href="../packages/metaRNASeq/index.html">metaRNASeq</a>
        meta-analysis from multiple RNA
      sequencing experiments,
<a href="../packages/MultiMeta/index.html">MultiMeta</a>
        for meta-analysis
      of multivariate GWAS
      results with graphics, designed to accept GEMMA format,
<a href="../packages/MetaSKAT/index.html">MetaSKAT</a>,
<a href="../packages/seqMeta/index.html">seqMeta</a>,
   provide meta-analysis
      for the SKAT test.
      </p>
      <h4>
        Interfaces
      </h4>
      <p>
<a href="../packages/RcmdrPlugin.EZR/index.html">RcmdrPlugin.EZR</a>
        provides an interface
    via the Rcmdr GUI
    using
<a href="../packages/meta/index.html">meta</a>
        and
<a href="../packages/metatest/index.html">metatest</a>
        to do the heavy lifting,
<a href="../packages/RcmdrPlugin.RMTCJags/index.html">RcmdrPlugin.RMTCJags</a>
        provides an interface
    for network meta-analysis using BUGS code,
    and
<a href="../packages/MAVIS/index.html">MAVIS</a>
        provides a Shiny
    interface using
<a href="../packages/metafor/index.html">metafor</a>,
<a href="../packages/MAc/index.html">MAc</a>,
<a href="../packages/MAd/index.html">MAd</a>, and
<a href="../packages/weightr/index.html">weightr</a>.
      </p>
      <h4>
        Simulation
      </h4>
      <p>
        Extensive facilities for simulation are provided in
<a href="../packages/metagen/index.html">metagen</a>
        including the ability to make use
of parallel processing.
<a href="../packages/psychmeta/index.html">psychmeta</a>
        provides
facilities for simulation of psychometric data-sets.
      </p>
      <h4>
        Others
      </h4>
      <p>
<a href="../packages/CRTSize/index.html">CRTSize</a>
        provides meta-analysis as part of a package
    primarily dedicated to the determination
    of sample size in cluster randomised trials in
    particular by simulating adding a new study to the
    meta-analysis.
      </p>
      <p>
<a href="../packages/CAMAN/index.html">CAMAN</a>
        offers the possibility of
    using finite semiparametric mixtures as an
    alternative to the random effects model
    where there is heterogeneity.
    Covariates can be included to provide meta-regression.
      </p>
      <p>
<a href="../packages/joineRmeta/index.html">joineRmeta</a>
        provides functions for meta-analysis of a single longitudinal and
   a single time-to-event outcome from multiple studies using joint models
      </p>
    </div>

  <h3>CRAN packages:</h3>
  <ul>
    <li><a href="../packages/aggregation/index.html">aggregation</a></li>
    <li><a href="../packages/altmeta/index.html">altmeta</a></li>
    <li><a href="../packages/bamdit/index.html">bamdit</a></li>
    <li><a href="../packages/bayesmeta/index.html">bayesmeta</a></li>
    <li><a href="../packages/bmeta/index.html">bmeta</a></li>
    <li><a href="../packages/bspmma/index.html">bspmma</a></li>
    <li><a href="../packages/CAMAN/index.html">CAMAN</a></li>
    <li><a href="../packages/CIAAWconsensus/index.html">CIAAWconsensus</a></li>
    <li><a href="../packages/clubSandwich/index.html">clubSandwich</a></li>
    <li><a href="../packages/compute.es/index.html">compute.es</a></li>
    <li><a href="../packages/ConfoundedMeta/index.html">ConfoundedMeta</a></li>
    <li><a href="../packages/CopulaREMADA/index.html">CopulaREMADA</a></li>
    <li><a href="../packages/CPBayes/index.html">CPBayes</a></li>
    <li><a href="../packages/CRTSize/index.html">CRTSize</a></li>
    <li><a href="../packages/diagmeta/index.html">diagmeta</a></li>
    <li><a href="../packages/dosresmeta/index.html">dosresmeta</a></li>
    <li><a href="../packages/ecoreg/index.html">ecoreg</a></li>
    <li><a href="../packages/effsize/index.html">effsize</a></li>
    <li><a href="../packages/epiR/index.html">epiR</a></li>
    <li><a href="../packages/esc/index.html">esc</a></li>
    <li><a href="../packages/etma/index.html">etma</a></li>
    <li><a href="../packages/exactmeta/index.html">exactmeta</a></li>
    <li><a href="../packages/extfunnel/index.html">extfunnel</a></li>
    <li><a href="../packages/forestmodel/index.html">forestmodel</a></li>
    <li><a href="../packages/forestplot/index.html">forestplot</a></li>
    <li><a href="../packages/gap/index.html">gap</a></li>
    <li><a href="../packages/gemtc/index.html">gemtc</a></li>
    <li><a href="../packages/getmstatistic/index.html">getmstatistic</a></li>
    <li><a href="../packages/gmeta/index.html">gmeta</a></li>
    <li><a href="../packages/hetmeta/index.html">hetmeta</a></li>
    <li><a href="../packages/ipdmeta/index.html">ipdmeta</a></li>
    <li><a href="../packages/joineRmeta/index.html">joineRmeta</a></li>
    <li><a href="../packages/joint.Cox/index.html">joint.Cox</a></li>
    <li><a href="../packages/MAc/index.html">MAc</a></li>
    <li><a href="../packages/MAd/index.html">MAd</a></li>
    <li><a href="../packages/mada/index.html">mada</a></li>
    <li><a href="../packages/MAVIS/index.html">MAVIS</a></li>
    <li><a href="../packages/MendelianRandomization/index.html">MendelianRandomization</a></li>
    <li><a href="../packages/meta/index.html">meta</a> (core)</li>
    <li><a href="../packages/meta4diag/index.html">meta4diag</a></li>
    <li><a href="../packages/MetaAnalyser/index.html">MetaAnalyser</a></li>
    <li><a href="../packages/MetABEL/index.html">MetABEL</a></li>
    <li><a href="../packages/metaBMA/index.html">metaBMA</a></li>
    <li><a href="../packages/metacart/index.html">metacart</a></li>
    <li><a href="../packages/metacor/index.html">metacor</a></li>
    <li><a href="../packages/metafor/index.html">metafor</a> (core)</li>
    <li><a href="../packages/metaforest/index.html">metaforest</a></li>
    <li><a href="../packages/metafuse/index.html">metafuse</a></li>
    <li><a href="../packages/metagear/index.html">metagear</a></li>
    <li><a href="../packages/metagen/index.html">metagen</a></li>
    <li><a href="../packages/metagen/index.html">metagen</a></li>
    <li><a href="../packages/MetaIntegrator/index.html">MetaIntegrator</a></li>
    <li><a href="../packages/metaLik/index.html">metaLik</a></li>
    <li><a href="../packages/metaMA/index.html">metaMA</a></li>
    <li><a href="../packages/metamisc/index.html">metamisc</a></li>
    <li><a href="../packages/metansue/index.html">metansue</a></li>
    <li><a href="../packages/metap/index.html">metap</a></li>
    <li><a href="../packages/MetaPath/index.html">MetaPath</a></li>
    <li><a href="../packages/MetaPCA/index.html">MetaPCA</a></li>
    <li><a href="../packages/metaplotr/index.html">metaplotr</a></li>
    <li><a href="../packages/metaplus/index.html">metaplus</a></li>
    <li><a href="../packages/MetaQC/index.html">MetaQC</a></li>
    <li><a href="../packages/metaRNASeq/index.html">metaRNASeq</a></li>
    <li><a href="../packages/metaSEM/index.html">metaSEM</a></li>
    <li><a href="../packages/metasens/index.html">metasens</a></li>
    <li><a href="../packages/MetaSKAT/index.html">MetaSKAT</a></li>
    <li><a href="../packages/metatest/index.html">metatest</a></li>
    <li><a href="../packages/Metatron/index.html">Metatron</a></li>
    <li><a href="../packages/metavcov/index.html">metavcov</a></li>
    <li><a href="../packages/metaviz/index.html">metaviz</a></li>
    <li><a href="../packages/mmeta/index.html">mmeta</a></li>
    <li><a href="../packages/MultiMeta/index.html">MultiMeta</a></li>
    <li><a href="../packages/mvmeta/index.html">mvmeta</a></li>
    <li><a href="../packages/mvtmeta/index.html">mvtmeta</a></li>
    <li><a href="../packages/netmeta/index.html">netmeta</a></li>
    <li><a href="../packages/nmaINLA/index.html">nmaINLA</a></li>
    <li><a href="../packages/nmathresh/index.html">nmathresh</a></li>
    <li><a href="../packages/pcnetmeta/index.html">pcnetmeta</a></li>
    <li><a href="../packages/pimeta/index.html">pimeta</a></li>
    <li><a href="../packages/psychmeta/index.html">psychmeta</a></li>
    <li><a href="../packages/psychometric/index.html">psychometric</a></li>
    <li><a href="../packages/PubBias/index.html">PubBias</a></li>
    <li><a href="../packages/RandMeta/index.html">RandMeta</a></li>
    <li><a href="../packages/ratesci/index.html">ratesci</a></li>
    <li><a href="../packages/RBesT/index.html">RBesT</a></li>
    <li><a href="../packages/RcmdrPlugin.EZR/index.html">RcmdrPlugin.EZR</a></li>
    <li><a href="../packages/RcmdrPlugin.RMTCJags/index.html">RcmdrPlugin.RMTCJags</a></li>
    <li><a href="../packages/revtools/index.html">revtools</a></li>
    <li><a href="../packages/rma.exact/index.html">rma.exact</a></li>
    <li><a href="../packages/rmeta/index.html">rmeta</a></li>
    <li><a href="../packages/robumeta/index.html">robumeta</a></li>
    <li><a href="../packages/SAMURAI/index.html">SAMURAI</a></li>
    <li><a href="../packages/SCMA/index.html">SCMA</a></li>
    <li><a href="../packages/selectMeta/index.html">selectMeta</a></li>
    <li><a href="../packages/seqMeta/index.html">seqMeta</a></li>
    <li><a href="../packages/surrosurv/index.html">surrosurv</a></li>
    <li><a href="../packages/TFisher/index.html">TFisher</a></li>
    <li><a href="../packages/weightr/index.html">weightr</a></li>
    <li><a href="../packages/xmeta/index.html">xmeta</a></li>
  </ul>

  <h3>Related links:</h3>
  <ul>
    <li>CRAN Task View: <a href="Genetics.html">Genetics</a></li>
    <li>CRAN Task View: <a href="ClinicalTrials.html">ClinicalTrials</a></li>
  </ul>

</body>
</html>
